src/components/clusterprovider/azure/azureclusterprovider.ts

```diff
@@ -283,6 +284,8 @@ async function createCluster(previousData: any, context: azure.Context): Promise
         } };
     const createResult = await azure.createCluster(context, options);
 
+    reporter.sendTelemetryEvent("createCluster", { result: createResult.result.succeeded ? "success" : "fail", clusterType: previousData.clusterType });
```

Comment:
Is there anything we can do to pre-empt GDPR classification for these new events and fields, or do we have to wait for them to roll in and the 'you have unclassified data' notifications to start?

Reply:
@itowlson We could trigger this telemetry by ourself when pr merged , so that the GDPR checker could find the new trace and tag the traces soon, and it seems that's all what we can do for GDPR.